### DISCUSSION TITLE
Add documentation link to Submission Attributes

### DISCUSSION THREAD

**Discussion by miguelverweij** [microsoft/PowerPlatform-Catalog#21] [Ideas]
If there would be an optional documentation attribute, we could link to a page where there is more information on the catalog item. Personally, I would add a Catalog section to the internal [Power Platform Hub site](https://learn.microsoft.com/en-us/power-platform/guidance/adoption/wiki-community), and for every catalog item create a page with information on what the catalog item does, installation instructions, feedback section, etc.

**Comment by MattB-msft** [comment]
There is support for a documentation link, source link and privacy policy.  
This is called "HelpLink" in the submission document

You can set them in the submission document as part of the catalog item.

**Comment by miguelverweij** [comment]
I see it. Got it working now. Sweet!
